terraform/modules/cms/cloudrun-geo.tf

```diff
@@ -0,0 +1,83 @@
+locals {
+  plateauview_geo_secret = []
+  plateauview_geo_ramdom = [
+    "REEARTH_PLATEAUVIEW_GEO_TOKEN",
+  ]
+}
+
+resource "google_cloud_run_service" "plateauview_geo" {
+  name                       = "plateauview-geo"
+  location                   = var.gcp_region
+  autogenerate_revision_name = true
+  metadata {
+    annotations = {
+      "run.googleapis.com/launch-stage"   = "BETA"
+      "run.googleapis.com/ingress"        = "all"
+      "run.googleapis.com/ingress-status" = "all"
+    }
+  }
+
+  template {
+    spec {
+      service_account_name = google_service_account.plateauview_geo.email
+      timeout_seconds      = 3600
+      containers {
+        # 初回作成時にreearth/reearthを指定すると、環境変数の設定不足で立ち上がらない。
+        # そのため、一時的にサンプルアプリケーションでで作成し、セットアップ完了後にgcloudでdeployを行う。
+        image = "gcr.io/cloudrun/hello"
+        resources {
+          limits = {
+            cpu    = "1000m"
+            memory = "512Mi"
+          }
+        }
+        ports {
+          container_port = 8080
+          name           = "h2c"
+        }
+
+        env {
+          name  = "REEARTH_GEO_GCS_BUCKETNAME"
+          value = google_storage_bucket.assets.name
+        }
+        env {
+          name  = "GOOGLE_CLOUD_PROJECT"
+          value = var.gcp_project_name
+        }
+      }
+    }
+    metadata {
+      annotations = {
+        "autoscaling.knative.dev/maxScale"         = "10"
+        "run.googleapis.com/execution-environment" = "gen2"
+      }
+    }
+  }
+  traffic {
+    percent         = 100
+    latest_revision = true
+  }
+  lifecycle {
+    ignore_changes = [
+      metadata[0].annotations,
+      template[0].spec[0].containers[0].image,
+      template[0].metadata[0].annotations["run.googleapis.com/client-name"],
+      template[0].metadata[0].annotations["client.knative.dev/user-image"],
+      template[0].metadata[0].annotations["run.googleapis.com/client-version"]
+    ]
+  }
+  depends_on = []
+}
+
+resource "random_string" "plateauview_geo_env" {
```

Comment:
Q. Should the created token be added to the environment variables?

Reply:
I thought it will be necessary in the future, but it's not sure. So I'll remove it for now.

Reply:
I fixed it:pray:
[Update cloudrun-geo.tf](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/5/commits/328df4c8e92cef0e31d0a09e383c3dfb930da324)